### PR TITLE
GUI update

### PR DESCRIPTION
**Pull request by CURIOUSER005**
Create host page arranged properly
Make Tables and forms of same format
Proper Indentation

### CODE REVIEW

File: satellite/templates/VM_info.html [hunk 1]
```diff
@@ -1,10 +1,7 @@
 {% extends 'base.html' %}
+{% block content %}
 
-{% block vmpackages %}
-<script src="/static/vm_packages.js"></script>
```
Comment (khomesh24) [inline satellite/templates/VM_info.html]: @CURIOUSER005 why did you remove /static/vm_packages.js

File: satellite/templates/VM_info.html [hunk 2]
```diff
@@ -16,6 +13,9 @@ <h2 class="card-pf-title">
                         {{ details.Name }}
                     </h2>
                     <div class="btn-group card-pf-title" style="float:right;top:-60px;">
+                        <!--<button class="btn btn-success">Start</button>-->
```
Comment (khomesh24) [inline satellite/templates/VM_info.html]: @CURIOUSER005 I have removed this comment long time ago

File: satellite/templates/VM_info.html [hunk 3]
```diff
@@ -37,12 +37,10 @@ <h2 class="card-pf-title">
                     </h2>
                 </div>
                 <div class="card-pf-body" id="vm_packages">
-                    <table class="table table-striped table-bordered table-hover" id="myPackages">
```
Comment (yadneshk) [inline satellite/templates/VM_info.html]: @CURIOUSER005 This id is required to search packages in VM packages table. Without this searching won't work

File: satellite/templates/VM_info.html [hunk 4]
```diff
@@ -37,12 +37,10 @@ <h2 class="card-pf-title">
...
+                    <table class="table table-striped table-bordered table-hover">
                         <thead>
                         <tr>
-                            <th>Package Name
-                            <input style="float:right" type="text" id="myInput" onkeyup="myFunction()" placeholder="Search for packages...">
-                            </th>
```
Comment (yadneshk) [inline satellite/templates/VM_info.html]: @CURIOUSER005 This textfield is needed to search packages. Where will the user enter the name of the package now?
